### PR TITLE
Fix cmake to find ispc compiler

### PR DESCRIPTION
**Pull request by alkino**
NEURON_BRANCH=master

### CODE REVIEW

File: CMake/CMakeTestISPCCompiler.cmake [hunk 1]
```diff
@@ -1,3 +1,4 @@
+include(CMakeTestCompilerCommon)
```
Comment (pramodk) [inline CMake/CMakeTestISPCCompiler.cmake:1]: could you confirm is this works with Cmake 3.8/3.9?

Comment (alkino) [reply CMake/CMakeTestISPCCompiler.cmake:1]: of course not :'(

**Comment by pramodk** [comment]
Retest this please

**Comment by alkino** [comment]
Please retest

**Comment by alkino** [comment]
Please retest

**Review by cattabiani — APPROVED** [review]

File: CMake/CMakeTestISPCCompiler.cmake (hunk 1 above)
Comment (pramodk) [reply CMake/CMakeTestISPCCompiler.cmake:1]: Does this fail with CMake v3.8? (If so, it shouldn't have merged?)

According to below, we say CMake 3.7 compatible: https://github.com/BlueBrain/CoreNeuron/blob/c03400580f6eeefdd4de994ade8aba609ac32042/CMakeLists.txt#L7 

We should implement this in CoreNEURON CI : https://github.com/neuronsimulator/nrn/pull/812

**Comment by alkino** [comment]
No it doesn't

File: CMake/CMakeTestISPCCompiler.cmake (hunk 1 above)
Comment (alkino) [reply CMake/CMakeTestISPCCompiler.cmake:1]: no it doesn't fail

Comment (alkino) [reply CMake/CMakeTestISPCCompiler.cmake:1]: what was failing was to use the new form of functions

Comment (pramodk) [reply CMake/CMakeTestISPCCompiler.cmake:1]: ok, thanks for clarification!

**Pull request merged**